app/components/pages-menu/component.js

```diff
@@ -4,6 +4,20 @@ export default Ember.Component.extend({
     style: Ember.computed('layer.settings.values.color', function(){
         return Ember.String.htmlSafe('color: ' + (this.get('layer.settings.values.color') || '#333') + '; ');
     }),
+    sticky: Ember.observer('layer.settings.values.StickToTop' , function() {
+        let topOfNav = $('.pages-menu').offset().top;
+        if(this.get('layer.settings.values.StickToTop')){
+            $(window).on('scroll', function (e) { 
+                if ( $(window).scrollTop() >= topOfNav ) {
+                    $('.pages-menu').addClass('sticky-nav')
+                }else{
+                    $('.pages-menu').removeClass('sticky-nav')
+                } 
+            });
+        }else{
+           $(window).off('scroll');
```

Comment:
This removes all scroll event handlers from window ,which may have breaking consequences for other code. Specify the function in off. Check documentation here: http://api.jquery.com/off/